colossalai/shardformer/policies/autopolicy.py

```diff
@@ -68,6 +68,14 @@ class PolicyLocation:
         PolicyLocation(file_name="gpt2", class_name="GPT2ForTokenClassificationPolicy"),
     "transformers.models.gpt2.modeling_gpt2.GPT2ForSequenceClassification":
         PolicyLocation(file_name="gpt2", class_name="GPT2ForSequenceClassificationPolicy"),
+
+    # OPT
+    "transformers.models.opt.modeling_opt.OPTModel":
+        PolicyLocation(file_name="opt", class_name="OPTModelPolicy"),
+    "transformers.models.opt.modeling_opt.OPTForCausalLM":
+        PolicyLocation(file_name="opt", class_name="OPTForCausalLMPolicy"),
+    "transformers.models.opt.modeling_opt.OPTForSequenceClassification":
+        PolicyLocation(file_name="opt", class_name="OPTForSequenceClassificationPolicy"),
```

Comment:
This does not cover all OPT models, these two are not included.
<img width="265" alt="Screenshot 2023-06-27 at 14 57 23" src="https://github.com/hpcaitech/ColossalAI/assets/31818963/98ca1ea5-5057-4e07-a183-a25fcffc5a93">